src/main/java/de/zalando/zmon/scheduler/ng/CommandSerializer.java

```diff
@@ -24,21 +21,43 @@ public CommandSerializer(TaskSerializerType type, Tracer tracer) {
     }
 
     public String expiresTime(long interval) {
-        Date exp = new Date(System.currentTimeMillis()+(interval * 1000L));
+        Date exp = new Date(System.currentTimeMillis() + (interval * 1000L));
```

Comment:
it's error-prone to have `interval` as `long`. E.g. if `interval == Long.MAX_VALUE`, what do you expect `interval * 1000` to be equal to? I'd change `interval` to `int` and maybe rename it some way to clarify it's in seconds...

Reply:
@alexkorotkikh  this piece of code was previously there. Only formatting changes were introduced by me. I understand your point. However, I would solve this in another PR rather than using this PR to solve it

Reply:
oh, I was inattentive. Sorry about that.